frontends/election-manager/src/screens/print_test_deck_screen.tsx

```diff
@@ -74,41 +68,26 @@ function PrecinctTallyReport({
 
 interface BmdPaperBallotsProps {
   electionDefinition: ElectionDefinition;
-  onAllRendered: (numBallots: number) => void;
   precinctId: PrecinctId;
 }
 
-function BmdPaperBallots({
+function getBmdPaperBallots({
   electionDefinition,
-  onAllRendered,
   precinctId,
-}: BmdPaperBallotsProps): JSX.Element {
+}: BmdPaperBallotsProps): JSX.Element[] {
```

Comment:
I know this adds complexity, but I chose to do it this way to separate this from the components of the test deck from the logic of printing on this particular screen. We already have a ticket to save the L&A package as a PDF #1903. In the interests of that, I wanted to have the components exist separately even if it required an intermediary. That way it will be very easy to use the `printElementToPdf` functionality to print elements to PDF - which we may want to chunk different if it is very large.

If we need to use a hook, we can wrap them in another component with a hook.

If you think this doesn't justify the complexity, I can revisit!

Reply:
Ah I hadn't thought PDF printing and making code reusable for that use case. I'm now better seeing the motivation for the current helpers, and it seems reasonable to keep them!